docs/HOW-TO-USE.md

```diff
@@ -43,14 +49,14 @@ As Vulnerability Scanning Tools use `sitemap.xml`, `robots.txt` etc. in order to
 
 ## Glimpse of the tool ##
 Tool's Starting Screen:
-![Welcome](/docs/Starting%20Screen.png)
+![Welcome](Starting%20Screen.png)
 
 On Clicking **Learn Security** following screen pops up:
-![Learning Security](docs/Vulnerability%20list.png)
+![Learning Security](Vulnerability list.png)
 This screen contains the List of Vulnerabilities, there information and useful links for learning more about them.
 
 On Clicking **Practice Vulnerability** following screen pops up:
-![Practice Vulnerability](docs/Choosing%20Vulnerability%20Level%20Screen.png)
+![Practice Vulnerability](Choosing Vulnerability Level Screen.png)
 This screen contains list of Levels present under the selected vulnerability, help at each level etc.
 
-#### Get Set Go for the Journey of Learning about the Web Application Security, MR. Security Enthusiast !!! ####
+Get Set Go for the Journey of Learning about the Web Application Security, MR. Security Enthusiast !!!
```

Comment:
this can be made bold. also please rewrite if not correct.

Reply:
I thought the text was good :smile:  So have just made it bold